### DISCUSSION TITLE
Welcome to ValvAndroidVault Discussions!

### DISCUSSION THREAD

**Discussion by hej2010** [Arctosoft/Valv-Android#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by XBladexEdgeX** [comment]
@hej2010 not sure if this would qualify as a bug report so i'm putting it here:
 after i updated to the latest version a few days ago, all my files have become inaccessible. i have multiple folders, and every thumbnail is just blank. i open the folder, and i can see the files being loaded because i see their names flashing by, but when it finishes loading, there's nothing there. i am unable to interact with my files in any other way than deleting the whole folder, i can't select anything to be exported. any help would be greatly appreciated, thank you.
**device: pixel 8**
**android version 14**

**Reply by hej2010** [reply]
Hi!
That happens when you input the wrong password. When the app can't decrypt the thumbnail it removes the file from the grid/list
